src/core/alt-table.rkt

```diff
@@ -3,6 +3,7 @@
 (require "../common.rkt")
 (require "../alternative.rkt")
 (require "../points.rkt")
+(require "../syntax/softposit.rkt")
```

Comment:
Where does `alt-table` need `softposits`? Also, the changes in the rest of this file seem to be either a bad merge or spacing changes. I think you should revert this to the version in `develop`.